template/build/core/phing/tasks/deploy.xml

```diff
@@ -83,6 +83,8 @@
   <target name="deploy:commit">
     <!-- We make these commands quiet because they can cause the output to be so long that Travis CI stops logging. -->
     <exec command="git add -A" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
+    <!-- We are forcing vendor to add here in case a global git ignore flags it -->
+    <exec command="git add vendor -f" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
```

Comment:
Took it one step further and added wildcards before and after so we capture ALL vendor directories everywhere. I say this because in a `/gitignore` file `/vendor` will refer to any vendor directory anywhere.